src/test/java/edu/cornell/kfs/fp/batch/service/impl/ProcurementCardCreateDocumentServiceImplTest.java

```diff
@@ -23,7 +25,7 @@
     private ProcurementCardCreateDocumentService procurementCardCreateDocumentService;
     private ConfigurationService  kualiConfigurationService;
     
-    private static org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ProcurementCardSummaryFeedService.class);
+    private static final Logger LOG = LogManager.getLogger(ProcurementCardSummaryFeedService.class);
```

Comment:
Should this logger be referencing the unit test class instead? (There appear to be several other unit test classes that also have this mismatched-class setup for their logging.)

Reply:
hmm, good question, I think it should probably be the unit test